packages/@stylexjs/postcss-plugin/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/core": "^7.26.8",
     "@stylexjs/babel-plugin": "0.13.0",
+    "@csstools/postcss-cascade-layers": "^5.0.1",
```

Comment:
This polyfill should be used by `processStylexRules` rather than the postcss plugin